core-kotlin-modules/core-kotlin-10/src/test/kotlin/com/baeldung/variableshadowing/VariableShadowingUnitTest.kt

```diff
@@ -0,0 +1,188 @@
+@file:Suppress("UNUSED_VARIABLE", "UNUSED_ANONYMOUS_PARAMETER")
+
+package com.baeldung.variableshadowing
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class VariableShadowingUnitTest{
```

Comment:
Can you run the Baeldung formatter over this class? I can see some formatting issues like this
```suggestion
class VariableShadowingUnitTest {
```